independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanInfo.java

```diff
@@ -932,7 +932,7 @@ private void doAddClassLevelBindings(ClassInfo classInfo, Collection<AnnotationI
             if (superClass != null) {
                 // proper interceptor binding inheritance only in strict mode, due to Quarkus expecting security
                 // annotations (such as `@RolesAllowed`) to be inherited, even though they are not `@Inherited`
```

Comment:
The 2 lines above need to be deleted now.

Reply:
Good point, done.